src/main/java/com/diffblue/deeptestutils/powermock/MemberMatcher.java

```diff
@@ -0,0 +1,48 @@
+package com.diffblue.deeptestutils.powermock;
+
+import java.lang.reflect.Method;
+
+/**
+ * <code>MemberMatcher</code> extends the original class in PowerMock
+ * and fixes the issues below.
+ * - TG-5254
+ *
+ * @author <a href="http://diffblue.com">Diffblue</a>
+ */
+public class MemberMatcher
+        extends org.powermock.api.support.membermodification.MemberMatcher {
```

Comment:
import org.powermock.api.support.membermodification.MemberMatcher

Reply:
In order to do that I have to rename the class, which probably isn't the worst thing anyway - I chose `DTUMemberMatcher`.